File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        
        int var = 0;

        for (int i=0; i<nums.length; i++){
            var = nums[i];
            for (int j=i + 1; j<nums.length; j++){
                if (nums[j] == var){
                    return true;
                }
            }
        }

        return false;


    }
}

File: Data Structures & Algorithms/minimum-difference-between-highest-and-lowest-of-k-scores/submission-1.py
class Solution:
    def minimumDifference(self, nums: List[int], k: int) -> int:


        nums.sort()

        minRes = nums[k-1] - nums[0]

        for i in range(k,len(nums)):

            res = nums[i] - nums[i-k+1]
            minRes = min(minRes, res)

        return minRes
            
